### PR TITLE
feat: azure blob integration

### PR DESCRIPTION
**Pull request by clowenhg**

### CODE REVIEW

File: integrations_test.go [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/rockset/rockset-go-client"
 	"github.com/rockset/rockset-go-client/option"
 )
 
+func (s *IntegrationsSuite) TestAzureBlob() {
+	s.T().Skip("pending better abstraction for local testing")
```
Comment (pmenglund) [inline integrations_test.go:19]: We need an Integration test for this, like we have for e.g. the S3 integration. I can help you get Azure credentials injected into the Circle CI environment.

File: integrations.go [hunk 2]
```diff
@@ -51,6 +51,29 @@ func (rc *RockClient) DeleteIntegration(ctx context.Context, name string) error
 	return err
 }
 
+func (rc *RockClient) CreateAzureBlobStorageIntegration(ctx context.Context, name string,
```
Comment (pmenglund) [inline integrations.go:54]: add a doc string

**Review by pmenglund — CHANGES_REQUESTED** [review]

File: integrations_test.go (hunk 1 above)
Comment (clowenhg) [reply integrations_test.go]: Filled the test out much like the S3 test, with obvious changes and use of the test suite

File: integrations.go (hunk 2 above)
Comment (clowenhg) [reply integrations.go:56]: added a doc string

**Comment by clowenhg** [comment]
Created a new branch/pr with this, that also adds testing.

**Pull request closed without merging**